modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/view/table/DateTimeClayTableSchemaField.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.data.set.view.table;
+
+/**
+ * @author Marcela Cunha
+ */
+public class DateTimeClayTableSchemaField extends DateClayTableSchemaField {
+
+	public DateTimeClayTableSchemaField() {
+		setContentRenderer("datetime");
+	}
+
+}
```

Comment:
This extension seems a bit off, since date+time and only date are two 'sibling' formats. Should we maybe have `BaseDateClayTableSchemaField`, extending into `DateClayTableSchemaField` and `DateTimeClayTableSchemaField`? I'm not sure it's that's Brian-kosher, we might end up with `DateDateClayTableSchemaField` and `DateTimeDateClayTableSchemaField`  :joy:

Reply:
Yes, maybe we can create a base class for them, or send this way and explain to Brian Chan in advance to know what he thinks :)